crates/ruff/src/rules/flake8_pytest_style/rules/raises.rs

```diff
@@ -83,11 +83,29 @@ fn is_pytest_raises(func: &Expr, semantic: &SemanticModel) -> bool {
     })
 }
 
+const fn is_compound_statement(stmt: &Stmt) -> bool {
+    matches!(
+        stmt,
+        Stmt::FunctionDef(_)
+            | Stmt::AsyncFunctionDef(_)
+            | Stmt::ClassDef(_)
+            | Stmt::While(_)
+            | Stmt::For(_)
+            | Stmt::AsyncFor(_)
+            | Stmt::Match(_)
+            | Stmt::With(_)
+            | Stmt::AsyncWith(_)
+            | Stmt::If(_)
+            | Stmt::Try(_)
+            | Stmt::TryStar(_)
+    )
+}
+
```

Comment:
https://github.com/astral-sh/ruff/blob/da33c26238d359e9e7a4d0549fc25b1e5c1ce7fc/crates/ruff_python_formatter/src/statement/suite.rs#L145

Reply:
I'll move it to `ruff_python_ast`...